src/components/Nav.svelte

```diff
@@ -35,7 +35,7 @@
 	}
 </style>
 
-<nav class="m-2 mb-24 px-1 flex md:justify-between flex-wrap justify-center">
+<nav class="p-4 flex flex-wrap justify-center md:justify-between bg-white">
```

Comment:
Looks like here are tabs used for indention, while in other files spaces are used.

Reply:
ah thanks, the sapper skeleton uses single quotes and tabs, what are the opposite settings of my vs code, so it got mixed up a bit. I ran the auto linting of prettier over each file.